e2e/tests/ui_tests/multiparty-ga-tests/1v2_same_solicitor_GeneralApplication_test.js

```diff
@@ -92,7 +92,7 @@ Scenario('GA for 1v2 Same Solicitor - Send application to other party journey',
       await I.login(config.judgeLocalUser);
     }
     await I.judgeRequestMoreInfo('requestMoreInfo', 'sendApplicationToOtherParty', gaCaseReference, 'no', 'sendApplicationToOtherParty');
-    await waitForGACamundaEventsFinishedBusinessProcess(gaCaseReference, 'MAKE_DECISION', config.applicantSolicitorUser);
+    await waitForGACamundaEventsFinishedBusinessProcess(gaCaseReference, 'AWAITING_RESPONDENT_RESPONSE', config.applicantSolicitorUser);
```

Comment:
this is not the right state as well. supposed to be APPLICATION_ADD_PAYMENT. please check @Azam-Hmcts